### ISSUE TITLE
Setup club ambassador profiles in Salesforce

### ISSUE THREAD

**Issue by shaun-technovation** [Iridescent-CM/technovation-app#5255]
We need to send club ambassador details to Salesforce, when:

- a new club ambassador registers
- a club ambassador profile is added to a mentor account

**Comment by dboyer** [comment]
@shaun-technovation Definitely! I was going to recommend it for this coming sprint. I've pulled it into the product backlog.

**Comment by dboyer** [comment]
Tested on QA:

- New club ambassadors had name and email address to SF and were listed as a club ambassador
- Chapter ambassador info was also still flowing to Salesforce as expected
- I went to the record for a mentor and added a club ambassador profile to their account. The contact exists in the SF sandbox with correct email, program participation has 3 entries- Mentor 2025 season, Club Ambassador 2025 season listed twice and have the exact same information just updated 3 minutes apart.

Perhaps the duplicate entry is similar to the problem reported in #5134

**Comment by dboyer** [comment]
Oh good point. I logged in as an admin and assigned Wild Kratts a club ambassador profile. I then logged in as Wild Kratts to make sure they were part of the 2025 season since they were only a participant in 2024.

**Comment by dboyer** [comment]
Tested again and confirmed:

- A new club ambassador profile is only being added once
- Adding a club ambassador profile to an existing mentor profile - one contact record in SF with two participant records - one for mentor and one for ambassador

It appears that duplicate records are no longer being added to the system.
